src/pymmcore_widgets/_mda/_util/_labeled_slider.py

```diff
@@ -0,0 +1,127 @@
+from qtpy import QtCore, QtGui, QtWidgets
+
+
+class QLabeledSlider(QtWidgets.QWidget):
```

Comment:
as mentioned elsewhere, would be good if we could use the superqt labeled slider and just add play buttons and stuff to that.  I can have a look at that:

https://pyapp-kit.github.io/superqt/widgets/qlabeledslider/